### DISCUSSION TITLE
When in a project can no longer select repository when creating an issue.

### DISCUSSION THREAD

**Discussion by jwhughes-work** [community/community#101955] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Hi 

Have noticed past few days that when I am in a project which is used for multiple repositories, when I go to create an issue from within the project (to-do/list view) I can no longer select which repository the issue is for and already has one selected which isn't changeable. I have looked to see if a default is set or if project needed to be linked to repos etc, but cant seem to fix it. 

If this is no longer possible I understand, but was very convenient to create issues from project board.

Thanks

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by JCCourt** [comment]
It seems like you’re experiencing an issue with creating issues within a specific repository from a project view in GitHub. 
According to the [GitHub Docs](https://docs.github.com/articles/creating-an-issue), when creating an issue, you should be able to select the repository where you want the new issue to be created at the top of the “Create new issue” dialog.

However, there’s a [discussion](https://github.com/orgs/community/discussions/6433) on GitHub about a similar issue where users are unable to link an issue to multiple repositories.

[Some users](https://github.com/orgs/community/discussions/6433) have suggested using one repository as the single place to create issues across all the repos.

**Reply by alfonsogarza** [reply]
@jwhughes-work is reporting that the first part `you should be able to select the repository where you want the new issue to be created at the top of the “Create new issue” dialog.` is broken, and is not regarding selecting multiple repositories.

**Comment by alfonsogarza** [comment]
I'm having the same issue, when trying to create an issue from our project we cannot longer select the repository.


<img width="434" alt="Screenshot 2024-01-31 at 12 50 33 PM" src="https://github.com/community/community/assets/1130949/b1811f0c-15eb-4648-9ef2-1e7a6c998ebd">

It always defaults to the same repo when my view has the following filters: `assignee:alfonsogarza  is:open ` or `is:open repo:"patio23/urbium-api3"`

<img width="321" alt="Screenshot 2024-01-31 at 12 50 37 PM" src="https://github.com/community/community/assets/1130949/d8f43d04-3112-4cd5-bc70-a4b969f2cdbf">

But if the view's results includes more than one repo and we have the tickets grouped by repo it works fine and pops the dialog to select a repo correctly.

**Comment by alfonsogarza** [comment]
@JCCourt Any updates on this or way to escalate? It really makes Projects much less useful and slows our workflows significantly.

**Comment by kcgrimes** [comment, marked as answer]
Working as intended now fyi

https://github.com/orgs/community/discussions/111387#discussioncomment-8912424

**Reply by jwhughes-work** [reply]
thank you very much :)
